static/oas/v1/sports/components/rosters.yml

```diff
@@ -63,18 +63,40 @@ components:
                 type: array
                 items: 
                   $ref: "../../users/users.yml#/schemas/User"
-    PostRosterResponse:
+                            
+    PostRosterRequest:
       type: object
       properties:
-        status_code:
-          $ref: "../../common/schemas.yml#/components/schemas/StatusCode200"
-        status_string:
-          $ref: "../../common/schemas.yml#/components/schemas/StatusString200"
-        request_id:
-          $ref: "../../common/schemas.yml#/components/schemas/RequestId"
-    PostRosterRequest:
+        captain_email:
+          type: string
+          description: Email associated to the captain
+          example: test1@test.com
+        player_emails: 
+          type: array
+          items: 
+            type: string
+          description: User emails associated with the players
+          example: 
+            - test2@test.com
+            - test3@test.com
+
+    RosterModel:
       type: object
       properties:
+        id:
+          type: integer
+          description: Id of the roster
+          example: 1
+        created_at:
+          type: string
+          format: date-time
+          description: Created time for the roster
+          example: "2024-06-06T23:41:00.352908346Z"
+        updated_at:
+          type: string
+          format: date-time
+          description: Updated time for the roster
+          example: "2024-06-06T23:41:00.352908346Z"
         captain_email:
```

Comment:
I do not think this is showing the most recent pull from the repository as this is no longer captain_email and player emails; instead, they should be captain_ID and player IDs.